tests/functional/adapter/test_backup_table.py

```diff
@@ -0,0 +1,78 @@
+import pytest
```

Comment:
Would it be worth bringing in some of the comments from the previous tests back to explain whats going on in each test? wonder if any contributors who might want to modify or work in this area might find changing up the test to be complicated if they are more used to something like pandas or ae side of things

Reply:
I can add that in. Basically all of the tests just look for the presence (or lack thereof) of the string "backup no" in the resulting ddl to determine if the config made it through the pipe. One of them makes sure it's in the right spot (loosely).

Reply:
Thanks for adding the newline here. For whatever reason, it just makes it much easier to picture what is happening (that you're reconstructing the file).